config/locales/en.yml

```
@@ -41,14 +41,17 @@ en:
     heading: Log in
     login: Log in
     cancel: Cancel
-    mfa_heading: Log in - One Time Password Request
-    mfa_description: Please enter your one time code from your authenticator app
+    mfa_heading: Enter multi-factor authentication code
+    mfa_description: Enter the code generated by your authentication app
     new_password_heading: Set up your new password
     new_password_description: This is first time you're logging in, please set your password.
   mfa_enrolment:
-    heading: Set up your MFA
+    heading: Set up multi-factor authentication
+    heading_code_entry: Enter code
+    code_entry: Enter the code generated by your authentication app.
+    description: To sign into your account you need to use an authentication app. Add the GOV.UK Verify Manage certificates service to your chosen app by scanning the barcode below. The barcode is valid for three minutes. 
```

```suggestion
    description: To sign into your account you need to use an authentication app. Add the GOV.UK Verify Manage certificates service to your chosen app by scanning the QR code below. The code is valid for three minutes. 
```

I know this in contradiction to the design, but barcode is a different thing to QR code, so we should call it correctly.